### DISCUSSION TITLE
DropdownMenu 组件下拉展示失效

### DISCUSSION THREAD

**Discussion by NEVERGIVEUP0922** [youzan/vant#7940] [Q&A]
DropdownMenu 组件下拉，在浏览器或者部分手机端后台放置一段时间，展示会失效

 
        <h3>
          
          <van-dropdown-menu
            active-color="#FF2D5D"
            :class="'dropdown-menu-actived'"
          >
            <van-dropdown-item
              :title="statusMenuTitle"
              v-model="currentStatus"
              :options="allstatus"
              @change="selectOrderStatus"
            />
          </van-dropdown-menu>
          
          <van-dropdown-menu
            :overlay="false"
            :class="saleManName != '全部' ? 'dropdown-menu-actived' : ''"
          >
            <van-dropdown-item
              :title="saleManName"
              ref="selectBdItem"
              @opened="showSalesMenOption"
            />
          </van-dropdown-menu>
          
          <van-dropdown-menu
            active-color="#FF2D5D"
            :class="currentSaleType != 0 ? 'dropdown-menu-actived' : ''"
          >
            <van-dropdown-item
              v-model="currentSaleType"
              :options="saleTypes"
              @change="selectSaleTypes"
            />
          </van-dropdown-menu>
        </h3>

**Comment by nbollard4** [comment]
Hello
